test/irb/test_cmd.rb

```diff
@@ -406,6 +406,57 @@ def test_irb_source
         ], out)
     end
 
+    def test_help
+      IRB.init_config(nil)
+      input = TestInputMethod.new([
+          "help 'String#gsub'\n",
+          "\n",
+        ])
+      IRB.conf[:VERBOSE] = false
+      IRB.conf[:PROMPT_MODE] = :SIMPLE
+      irb = IRB::Irb.new(IRB::WorkSpace.new(self), input)
+      IRB.conf[:MAIN_CONTEXT] = irb.context
```

Comment:
Are setting these `IRB.conf` needed? The tests seems to pass even when they're commented out.